cvat/apps/documentation/user_guide.md

```diff
@@ -417,69 +577,88 @@ In a trivial case a correct filter should correspond to the template: ``//label[
+``H``                  | hide active shape
+``T+H``                | hide all shapes
 ``J``                  | hide labels with attributes on every frame
-``Delete``             | delete a selected object
-``Shift+Delete``       | delete a selected object even if it is locked
+``Delete``             | delete an active shape
+``Shift+Delete``       | delete an active shape even if it is locked
 ``F``                  | go to next frame
 ``D``                  | go to previous frame
 ``V``                  | go forward with a predefined step
 ``C``                  | go backward with a predefined step
-``Ctrl+C``             | copy a selected object
-``Ctrl+V``             | insert a copied object
-``F1``                 | open help
-``F1`` in dashboard    | open page with documentation
-``F2``                 | open settings
+``~``                  | focus to ``go to frame`` element
+``Ctrl+C``             | copy an active shape
+``Ctrl+V``             | insert a copied shape
+``Ctrl+Z``             | undo previous action
+``Ctrl+Shift+Z``/``Ctrl+Y``| redo previous action
+``Shift+B``/``Alt+B``  | increase/decrease brightness on an image
+``Shift+C``/``Alt+C``  | increase/decrease contrast on an image
+``Shift+S``/``Alt+S``  | increase/decrease saturation on an image
 ``Ctrl+S``             | save job
+``Ctrl+B``             | propagate active shape
 |                      | __Interpolation__             |
 ``M``                  | enter/apply merge mode
-``Ctrl+M``             | leave merge mode without saving changes
-``R``                  | go to the next key frame of a selected object
-``E``                  | go to the previous key frame of a selected object
+``Alt+M``              | close merge mode without apply the merge
+``R``                  | go to the next key frame of an active shape
+``E``                  | go to the previous key frame of an active shape
 |                      | __Attribute annotation mode__ |
 ``Shift+Enter``        | enter/leave Attribute Annotation mode
 ``Up Arrow``           | go to the next attribute (up)
 ``Down Arrown``        | go to the next attribute (down)
 ``Tab``                | go to the next annotated object
 ``Shift+Tab``          | go to the previous annotated object
 ``<number>``           | assign a corresponding value to the current attribute
+|                      | __Grouping__                  |
+``G``                  | switch group mode
+``Alt+G``              | close group mode
+``Shift+G``            | reset group for selected shapes
 |                      | __Filter__                    |
 ``Left Arrow``         | go to the previous frame which corresponds to the specified filter value
 ``Right Arrow``        | go to the next frame which corresponds to the specified filter value
-``K``                  | hide all objects which don't correspond to the specified filter value
+``K``                  | hide all shapes which don't correspond to the specified filter value
+
```

Comment:
I don't see shortcuts for z-order changing in the table. It has been added to shortcut table if task has 'z_order' == true flag.